### ISSUE TITLE
Mobile 'Home' Navigation Component

### ISSUE THREAD

**Issue by mikhael28** [mikhael28/paretOS#55]
A good chunk of the OS, like many before it, was designed for desktop experiences. However, for modules like the Arena - the user's will at least half the time interact with it on a mobile device. The biggest bottleneck that we see here is the navigation system - right now we have a BottomNav bar that pops up in tablet/mobile dimensions, but there are so many items/modules in the OS now/in the future to come that there is no way we can squeeze all of those items in. I would suggest an incredibly simple, Windows 7 Mobile (remember that?!) scrolling navigation component, with blocks/cards/links to your sprints, the library of context, to your curriculum, to leaderboards, to profiles - etc. 

The long story short, is we need to make navigation on mobile MUCH more intuitive. Perhaps in the future, we can even have people customize the layout of their Home Screen and choose what's on top, what color the block/card is, etc. Here is a screenshot that has a rough idea of what we are looking for, but different - obviously. 
![windows7-mobile](https://user-images.githubusercontent.com/15205259/131594241-0a375e7e-5584-482b-b811-1f37a7a917df.jpg)

Does anyone have any UI inspiration to contribute to this?

**Comment by jeremydthomas** [comment]
I'll give this a whirl

**Comment by mikhael28** [comment]
@jeremydthomas any progress? I think based on our chat in Discord, you were busy with work correct?
